CONTRIBUTING_CODE.md

```diff
@@ -1,51 +1,51 @@
 # Contributing to the Calico Codebase
 
+Thanks for considering contributing to Calico! This document outlines the canonical procedure for contributing new features
+and bugfixes to Calico - following these steps will help ensure that your contribution gets merged quickly and
```

Comment:
I would make two sentences: "Calico - following" --> "Calico. Following"

Otherwise (nit): I guess just make sure you have a real em dash character here and should be no spaces around it.